internal/views/details.go

```diff
@@ -111,6 +113,16 @@ func (v *detailsView) saveCmd(evt *tcell.EventKey) *tcell.EventKey {
 	return nil
 }
 
+func (v *detailsView) cpCmd(evt *tcell.EventKey) *tcell.EventKey {
+	log.Debug().Msgf("Copied YAML to clipboard")
```

Comment:
one nit request: details view is the same for `YAML` and `Describe`. But `Describe` is not an actual YAML, so we could change the message to `Copied content to clipboard`.

Reply:
Ah, didn't realize that. Done!

Reply:
I removed the debug log too, as it felt a bit unnecessary

Reply:
That's great! Thanks @syvanpera. Now let's wait for Fernand.